src/testrel.jl

```diff
@@ -470,6 +472,19 @@ function _test_rel_steps(;
     end
 end
 
+function check_flaky(name::String, logs::Vector{LogRecord})
+    retries = 0
+    for log in logs
+        if haskey(log.kwargs, :submit_failed) && log.kwargs[:retry_number] > retries
+            retries = log.kwargs[:retry_number]
```

Comment:
How is `:retry_number` passed through to the logger kwargs thing here?

Reply:
in the log's metadata